src/main/java/hudson/plugins/sloccount/SloccountBuildAction.java

```diff
@@ -47,10 +45,18 @@ public String getUrlName() {
     }
     
     @Override  
-    public Collection<? extends Action> getProjectActions() {  
-        return this.projectActions;  
-    }  
+    public Collection<? extends Action> getProjectActions() {
+        return this.projectActions;
```

Comment:
Probably should be wrapped with `Collections.unmodifyableList()`. Otherwise FindBugs may start grumbling if you set up a Low threshold

Reply:
Maybe also makes sense to check for null just to prevent NPE issues with exotic usages of the class via API (e.g. in Groovy Build Steps)